avogadro/qtplugins/focus/focus.cpp

```diff
@@ -0,0 +1,111 @@
+/******************************************************************************
```

Comment:
Drop all this. As we edit files, we're going to a new top comment:
```
/******************************************************************************
  This source file is part of the Avogadro project.
  This source code is released under the 3-Clause BSD License, (see "LICENSE").
******************************************************************************/
```

(In particular, you're editing this in 2022, not 2012 .. and you're not working for Kitware.)

Reply:
Ooops... Yes, I'm sorry, I just copied the notice from a different file. I'm pretty stupid when it comes to legal code.